Docker/pytools/Dockerfile

```diff
@@ -15,9 +15,7 @@ USER admin
 
 ENV HOME=/home/admin \
     PYENV_ROOT=$HOME/.pyenv \
-    PATH=$HOME/.local/bin:/home/admin/.pyenv/shims:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
-    PATH=$PATH:/home/admin/.pyenv/bin:$PYENV_ROOT/plugins/pyenv-virtualenv/shims \
-    PATH=$PATH:$HOME/.pyenv/shims:$HOME/bin:/$HOME/.poetry/bin \
+    PATH=/home/admin/.pyenv/shims:/home/admin/.pyenv/bin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/admin/.poetry/bin:. \
```

Comment:
I think that environment variable PATH should be assigned in the following way: PATH=/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/admin/.poetry/bin:. \

Because we are not installing any version of python, it should not contain any reference to pyenv inside Dockerfile, we only add this kind of reference to pyenv when we're gonna install any version of python or configure something of pyenv such as nvim does.

Reply:
Yes, but you are using pyenv , It will mark an error if I modify the way you indicate.
 
![image](https://user-images.githubusercontent.com/33498584/118031220-eb71c700-b32b-11eb-9191-1701f1f5eb26.png)

Reply:
You got it right 👍